internal/engine/executor_test.go

```diff
@@ -461,6 +461,53 @@ func TestMutatorRunInTheCorrectFolder(t *testing.T) {
 	})
 }
 
+func TestMutatorRunWithCorrectEnvs(t *testing.T) {
```

Comment:
Maybe this test can be integrated in the `TestMutatorTestExecution`, they are almost identical.